main.go

```diff
@@ -91,9 +92,9 @@ func getEnvKey(key string, required bool) string {
 	value, exist := os.LookupEnv(key)
 	if !exist {
 		if required {
-			log.Fatal("No provided environment variable for " + key)
```

Comment:
Why have you changed it?

Reply:
it gave me an error but before i update the libraries